packages/core/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  root: true,
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/@typescript-eslint',
+    'plugin:import/typescript',
+    'plugin:mocha/recommended',
+  ],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint', 'prettier', 'mocha'],
+  rules: {
+    quotes: [2, 'single'],
+    'object-curly-spacing': [2],
```

Comment:
Please correct these eslint rules (and fix corresponding errors), and I'm fine with the rest

Reply:
Propose:

```
    quotes: ['error', 'single'],
    'object-curly-spacing': ['error', 'always', { arraysInObjects: false }],
```